curiouslearning/FeedTheMonsterJS
# fix: feedback audio playing in levelendend scene (FM-341)

# Changes
- feedback audio playing in levelendend scene

# How to test
- Open the start screen of FTM.
- Tap or click any part of the start screen to proceed to the next screen
- Select any level
- Finish the game
- Make sure the feedback audio is not playing in levelend scene

Ref: [FM-341](https://curiouslearning.atlassian.net/browse/FM-341)


[FM-341]: https://curiouslearning.atlassian.net/browse/FM-341?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
